src/Native/pkg/dir.props

```diff
@@ -31,12 +31,15 @@
     <DebianNativePath Condition="'$(DebianNativePath)' == ''">$(BuildNativePath)</DebianNativePath>
     <Fedora23NativePath Condition="'$(Fedora23NativePath)' == ''">$(BuildNativePath)</Fedora23NativePath>
```

Comment:
can we remove references to old unsupported distros here at the same time?

Reply:
I'm more in favor of leaving them in until I get the new distros building successfully. That way if the build fails along the way I have fewer potential causes to investigate.

These should be removed after that point, though.

Reply:
nevermind, the problem isn't as severe as I first thought.  We can leave this to reduce risk / issues.

https://github.com/dotnet/core/blob/master/release-notes/1.0/1.0-supported-os.md#out-of-support-os-versions